src/CMakeLists.txt

```diff
@@ -22,3 +22,8 @@ if(NOT WIN32)
     set_target_properties(hera PROPERTIES LINK_FLAGS "-Wl,-undefined,error")
   endif()
 endif()
+
+if(HERA_WAVM)
+    target_compile_definitions(hera PRIVATE HERA_WAVM=1)
+    target_link_libraries(hera PRIVATE wavm::runtime)
```

Comment:
In addition to the Runtime lib (and the dependencies pulled in implicitly) we also need the WASM library to convert the bytecode into WAVM's internal IR.